### ISSUE TITLE
The real challenge that is awaiting is…

### ISSUE THREAD

**Issue by hadronized** [Geal/nomfun#4]
…can it be as fast as pest?

![](https://phaazon.net/media/uploads/goodenough.jpg)

**Comment by kpp** [comment]
Why? Is pest faster than nom?

**Comment by hadronized** [comment]
There’s an old legend of o’ folks stating it’s even twice faster!

**Comment by kpp** [comment]
Yeah an old legend. Whereas on https://pest.rs/ pest is said to be 2.44 times slower while processing JSON.

**Comment by hadronized** [comment]
![](https://phaazon.net/media/uploads/goodenough.jpg)

It’s all a running gag, nom isn’t slow. Also, I don’t really like to compare those as nom is a [scannerless parser](https://en.wikipedia.org/wiki/Scannerless_parsing) and pest is a real lexer that requires another pass of parsing (which I find dubious) to be complete. Some people attempted to fill that gap but I still haven’t seen anything stable yet.
